packages/inquirer/lib/prompts/checkbox.js

```diff
@@ -226,9 +226,12 @@ function renderChoices(choices, pointer) {
       output += ' - ' + choice.name;
       output += ' (' + (_.isString(choice.disabled) ? choice.disabled : 'Disabled') + ')';
     } else {
-      var isSelected = i - separatorOffset === pointer;
-      output += isSelected ? chalk.cyan(figures.pointer) : ' ';
-      output += getCheckbox(choice.checked) + ' ' + choice.name;
+      var line = getCheckbox(choice.checked) + choice.name;
```

Comment:
When rewriting this line, the blank space between the checkbox and the option name has been removed (e.g. `[]Choice A` vs `[] Choice A`)

Reply:
My mistake. Blank space was restored in the latest commit.